pages/search_page.py

```diff
@@ -28,7 +31,7 @@ def add_item_to_cart(self):
         self.click(by_locator=self.__ADD_BUTTON_FOR_SEARCH_RESULT_ITEM)
```

Comment:
this `ActionChains` can be extracted to a separate method to a base class and reused in case of need in many other places

Reply:
you can call thismethod e.g. `move_mouse_over(by_locator) and pass element locator to it`